DataFormats/L1TotemRP/interface/TotemRPCCBits.h

```diff
@@ -0,0 +1,77 @@
+/****************************************************************************
+ *
+ * This is a part of TOTEM offline software.
+ * Authors
+ *  Leszek Grzanka (braciszek@gmail.com)
+ ****************************************************************************/
+
+#ifndef DataFormats_L1TotemRP_TotemRPCCBits
+#define DataFormats_L1TotemRP_TotemRPCCBits
+
+#include <iosfwd>
+#include <iostream>
+#include <cstdlib>
+#include <bitset>
+
+/**
+ * TODO: describe
+ **/
+class TotemRPCCBits
+{
+  public:
+    /// Construct from a packed id. It is required that the Detector part of
+    /// id is Totem and the SubDet part is RP, otherwise an exception is thrown.
+    explicit TotemRPCCBits(uint32_t id, std::bitset<16> bs) : id_(id)
```

Comment:
how is this used?
Do we expect multiple TotemRPCCBits for the same ID?
Is this "id" related to TotemRPDetId?

It looks like id_ should be removed from the data members and the object collection should be stored in DetSetVector.
Code in  RawToDigiConverter::RPDataProduce #13839 is filling TotemRPDetId here.